### ISSUE TITLE
Text becomes unreadable as when it scrolls over images at higher zoom levels

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/hourglass#242]
## Report Issue
**Describe the inaccessible feature**
paralax means text ends up on top of an image when scrolling, such that contrast ratios are not maintained, including back text ending up on a black part of the image.

**Which users are impacted?**
 (delete as applicable)
 1. Everyone
 
**Assistive software or customised computer setup**
this appears to be a higher zoom issue, where at lower zoom the two do not make contact.

**To Reproduce**
Steps to reproduce:
1. Go to ['homepage'](https://olhbeta.janeway.systems/)
2. have a zoom of 175% with a window width of 660px (this may occur at other settings too)
3. Scroll down to the "our mission" section

**Screenshots**
<img width="407" alt="image" src="https://github.com/BirkbeckCTP/hourglass/assets/5330770/e8c40345-03db-42dd-9574-871906c781bc">

## Preferred Solution
Style the text background colour the same as the page background.  When directly against the page background this will not make a difference, but then when it scrolls up over an image, it will carry that background colour with it.

## Front-end Issues (only)
If the issue is front-end specific please add the following details:

**Desktop (please complete the following information):**
Details from Chrome on MacOS, but also seen in Safari, depends on the zoom/window width such that the mission text is below the image, where at lower zoom / wider windows it appears to the right of the image.

**Comment by StephDriver** [comment]
There has been general discussion of this issue in regard to moving towards thinking about colours in pairs (background and foreground) so that text is always styled with both a background and foreground colour.  And then when the background is the same as the page background nobody is any the wiser, but if the unexpected happens and it is not, the text remains readable.

There was further work between @joemull and myself on whether we could take this up a level and keep the styling consistent with the site branding by putting shapes behind text (same colour as background) rather than just rectangular background to the text as would happen by default. .  Again if the text is against the page background there would be no visible change, but it would mean regardless of how it reflows or sizes it would look styled and be readable, where the first solution only makes it readable.

**Comment by joemull** [comment]
@StephDriver thanks for remembering this and noting it here. I think we have it in design meeting running notes but good to have it here.

**Comment by mauromsl** [comment]
Joe to handle this as part of https://github.com/orgs/openlibhums/projects/91
